owners/OWNERS.example

```diff
@@ -131,11 +131,11 @@
   ],
 
   // The root-level owners file--and ONLY the root-level owners file--may
-  // specify an optional `reviewerTeam` property, providing the name of a GitHub
+  // specify an optional `reviewerPool` property, providing the name of a GitHub
```

Comment:
nit: Update comment: _"providing the name of a GitHub team"_ -> _"providing a list of GitHub team names and / or IDs"_. Also update the sentence that says _"one member of this team has given approval"_.

Reply:
Done